src/components/Garden/Feed/AbstainCard.tsx

```diff
@@ -1,12 +1,11 @@
 import React, { useCallback } from 'react'
 import { useHistory } from 'react-router-dom'
-import { GU, useTheme, useViewport } from '@1hive/1hive-ui'
```

Comment:
What do you think of removing the `Request: 0 TKN` from every other Suggestion proposal aswell?

Reply:
WDYT @0xGabi 

![Captura de ecrã 2022-01-23, às 12 33 02](https://user-images.githubusercontent.com/13184907/150678599-2c00808e-9cac-499a-aab7-ff289f68620b.png)

Reply:
Yes, looking much clear. Thanks!